resources/ts/Components/IntegrationCard.tsx

```diff
@@ -20,6 +21,25 @@ const productTypeToPath = {
   widgets: "/widgets/aan-de-slag",
 };
 
+const OpenWidgetBuilderButton = ({ id, type }: Pick<Props, "id" | "type">) => {
+  const { t } = useTranslation();
+  if (type !== "widgets") {
+    return null;
+  }
+
+  return (
+    <ButtonSecondary>
+      <a
```

Comment:
It would be better to use a link styled as a secondary button.
`ButtonLinkSecondary` exists if I'm not mistaken